apps/diet-app/src/app/types/types.ts

```diff
@@ -0,0 +1,8 @@
+export type WheelPickerProps = {
```

Comment:
It's decent place to put specific component props. I suggest to put it near the implementation of WheelPicker (the same file and inside the same folder) Adding it there creates blurry structure - types inside app router and component implementation is inside client/components